views/lib/csrf_content.hbs

```diff
@@ -1,8 +1,8 @@
 <div class="error-header-wrapper pb-3">
-    {{> 'lib/components/svg_error_content' }}
-    <h1 class="h2 error-header-code">CSRF</h1>
+    {{> 'lib/components/svg_error_to_much_tabs' }}
+    {{!-- <h1 class="h2 error-header-code">CSRF</h1> --}}
     <p class="error-header-text">
-        {{$t "lib.csrf_content.text.yourRequestCouldNotBeVerified" }}
+        {{$t "lib.csrf_content.text.yourSessionHasExpired" }}
```

Comment:
why not use the yourRequestCouldNotBeVerified anymore? The session is not technically expired (timed out), but denied for a different reason.

In case you really want to change the message here, please also remove the old text from the language files if its no longer used.